app/jmeter/jira.jmx

```diff
@@ -3061,12 +3061,12 @@ if ( sleep_time &gt; 0 ) {
                 <hashTree/>
                 <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
                   <collectionProp name="Asserion.test_strings">
-                    <stringProp name="1182867844">[&quot;project-key&quot;]=&quot;\&quot;${project_key}\&quot;&quot;;</stringProp>
+                    <stringProp name="-1259658522">.*project-key.*${project_key}</stringProp>
                   </collectionProp>
                   <stringProp name="Assertion.custom_message"></stringProp>
                   <stringProp name="Assertion.test_field">Assertion.response_data</stringProp>
                   <boolProp name="Assertion.assume_success">false</boolProp>
-                  <intProp name="Assertion.test_type">16</intProp>
+                  <intProp name="Assertion.test_type">2</intProp>
```

Comment:
What is this 16 and 2?

Reply:
It is type of "matching" rules.
Like 2 - "Contains"
16 - "Substring", etc.